hd-wallet/plugin.lua

```diff
@@ -1,440 +1,418 @@
 -- Name: HD Wallet
--- Version: 1.1
--- Description:##
--- This plugin implements hierarchical deterministic wallets (or "HD Wallets") BIP0032 protocol.
--- 
--- ### ## Introduction
--- The plugin allows to derive child key (xprv, xpub) from a master key in a deterministic way, and/or sign transaction hashes for UTXO and ethereum type crypto coins.
--- 
+-- Version: 1.3
+-- Description:
+-- This plugin implements hierarchical deterministic wallets (or "HD Wallets")
+-- BIP0032 protocol.
+--
+-- ## Introduction
+-- The plugin derives a child key in a given path from a master key, and signs
+-- a transaction hash. The child key is transient; it only exists during the
+-- plugin execution. This version of the plugin requires the master key to be
+-- exportable. In upcoming version 2.0, this condition is removed for better
+-- security.
+--
 -- ## Use cases
--- 
--- The plugin can be used to
--- 
--- - Derive child key for UTXO
```

Comment:
On the new version, the child private key is not returned to the caller, so this is removed.